kmod/src/inode.c

```diff
@@ -1417,7 +1418,7 @@ static void init_orphan_key(struct scoutfs_key *key, u64 rid, u64 ino)
 	};
 }
 
-static int remove_orphan_item(struct super_block *sb, u64 ino)
+int remove_orphan_item(struct super_block *sb, u64 ino)
```

Comment:
If it's exported it should start with scoutfs_.  To make it clear it's just a wrapper around setting a key and calling an item func, I'd have the naming match the item funcs.. so like scoutfs_orphan_delete() here.  ( and _dirty() for that dirty wrapper ).